packages/gnosis-safe/src/index.ts

```diff
@@ -106,7 +119,10 @@ export class GnosisSafe extends Connector {
         accounts: [await this.sdk!.safe.getInfo().then(({ safeAddress }) => safeAddress)],
       })
     } catch (error) {
-      this.actions.reportError(error as Error | undefined)
+      if (error) {
```

Comment:
is this right?

Reply:
i think even if error is undefined for whatever reason we still want to reset state, will make the change